Shared/en.lproj/Localizable.strings

```diff
@@ -496,10 +491,10 @@
 "Menu.CopyURL.Confirm" = "URL Copied To Clipboard";
 
 /* A switch to disable enhanced tracking protection inside the menu. */
-"Menu.EnhancedTrackingProtectionOff.Title" = "Enhanced Tracking Protection is OFF for this site.";
+"Menu.EnhancedTrackingProtectionOff.Title" = "Protections are OFF for this site";
```

Comment:
This was [changed here](https://github.com/mozilla-mobile/firefox-ios/commit/3fcd03774fa89119782c36c030f11163220e2a6d#diff-b743e29e051f39265ce2645f435dc519504660e0aa4e4ba5bad75b26b0e4f400).

The screenshots still show the old string for `en`, I wonder what's in the app actually shipping to users…

<img width="375" alt="iPhone 8-TrackingProtectionDisabledPerSite-02" src="https://user-images.githubusercontent.com/3644868/186634446-f1f48cd6-2107-4151-8dc0-3fcdbab30dc3.png">